offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/update/UpdatePersonView.groovy

```diff
@@ -63,7 +63,8 @@ class UpdatePersonView extends CreatePersonView{
         addAffiliationButton = new Button("Add Affiliation")
         addAffiliationButton.setIcon(VaadinIcons.PLUS)
         addAffiliationButton.setEnabled(false)
-
+        titleField.setRequiredIndicatorVisible(true)
+        titleField.setEmptySelectionAllowed(false)
```

Comment:
This is inherited from the create person view. There is no need to overwrite this.

Reply:
Ah you're right good catch 👁️